### ISSUE TITLE
"Double-sync" on local changes

### ISSUE THREAD

**Issue by Xyaneon** [Xyaneon/grive-daemon#9]
When local changes are made in common situations (i.e., saving a file), grive-daemon may initiate two grive syncs, one after another. This is due to it only seeing one inotify event right away, then syncing again once it sees the other ones. It would be better if there was a small delay to make sure all changes associated with a simple operation are accounted for before calling grive.

**Comment by ozomer** [comment]
I am not sure this is the same problem, but on my ubuntu 16.04 the script in `/etc/profile.d` caused multiple processes of grive-daemon to run in parallel. I am not sure which scenarios caused it - maybe creation of new terminals?
Anyways, I fixed it by removing the start-grive-daemon script from `/etc/profile.d` and added a new job to Ubuntu's "Startup Applications Preferences" that calls:
`sh -c 'sleep 10; grive-daemon'`

**Comment by Xyaneon** [comment]
@ozomer That would be a separate issue from this one. I went ahead and filed it as issue #13 for you. Thanks!
